warehouse/models/mart/gtfs_guidelines/fact_daily_transitland_url_check.sql

```diff
@@ -0,0 +1,6 @@
+WITH gtfs_fact_daily_transitland_url_check AS (
+    SELECT *
```

Comment:
an opinion that Andrew has shared that I think I agree with is that it is preferable to specifically enumerate columns rather than `SELECT *`, especially for mart tables, just so it's 100% clear what we expect to be getting. in this case where it's basically identical to the staging table it's a little silly but I think it's a good practice in general